met-api/src/met_api/models/survey.py

```diff
@@ -89,3 +96,17 @@ def update_survey(cls, survey: SurveySchema) -> DefaultMethodResult:
         query.update(update_fields)
         db.session.commit()
         return DefaultMethodResult(True, 'Survey Updated', survey_id)
+
+    @classmethod
+    def link_survey(cls, survey_id, engagement_id) -> DefaultMethodResult:
+        """Update engagement."""
```

Comment:
wrong docstring

Reply:
noted, will follow this up with change in a new PR